lms/djangoapps/support/static/support/js/spec/views/enrollment_spec.js

```diff
@@ -3,34 +3,34 @@ define([
     'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers',
     'support/js/spec_helpers/enrollment_helpers',
     'support/js/views/enrollment'
-], function (_, AjaxHelpers, EnrollmentHelpers, EnrollmentView) {
+], function(_, AjaxHelpers, EnrollmentHelpers, EnrollmentView) {
     'use strict';
 
     var enrollmentView,
-        createEnrollmentView = function (options) {
+        createEnrollmentView = function(options) {
             if (_.isUndefined(options)) {
                 options = {};
             }
             return new EnrollmentView(_.extend({}, {
                 el: '.enrollment-content',
                 user: 'test-user',
                 enrollmentsUrl: '/support/enrollment/',
-                enrollmentSupportUrl: '/support/enrollment/',
+                enrollmentSupportUrl: '/support/enrollment/'
```

Comment:
There are quite a few of these fixes that will be unnecessary when we switch the linter to allow trailing commas on multiple lines. It might be worth holding off until that's in place.